src/griffe/docstrings/google.py

```diff
@@ -519,8 +571,14 @@ def parse(  # noqa: WPS231
     in_code_block = False
     lines = docstring.lines
 
+    default_parser_options = {
+        "ignore_init_summary": False,
+        "trim_doctest_flags": True,
+    }
+    options = {**default_parser_options, **options}
```

Comment:
Since I'd like to keep options in the signature, this can be changed to something like:

```python
options = {
    "ignore_init_summary": ignore_init_summary,
    "trim_doctest_flags": trim_doctest_flags,
}
```